src/game/client/components/maplayers.cpp

```diff
@@ -71,6 +71,13 @@ CMapLayers::CMapLayers(int Type, bool OnlineOnly)
 	m_Params.m_RenderTileBorder = true;
 }
 
+void CMapLayers::Unload()
+{
+	for(auto &&It : m_vRenderLayers)
+		It->Unload();
```

Comment:
Was the rvalue reference necessary? Otherwise

```suggestion
	for(auto &Layer : m_vRenderLayers)
		Layer->Unload();
```

Reply:
Should I rename m_vRenderLayers already here? Conceptually when I named this variable I wasn't sure how to name it, since smart pointers are technically objects. Now I know better